content/client-server-api/modules/report_content.md

```diff
@@ -29,3 +29,20 @@ is in before accepting a report.
 based on whether or not the reporting user is joined to the room. This is
 because users can be exposed to harmful content without being joined to a
 room. For instance, through room directories or invites.
+
+{{% added-in v="1.14" %}} Similarly, servers MUST NOT restrict user reports
+based on whether or not the reporting user is joined to any rooms that the
+reported user is joined to. This is because users can be exposed to harmful
+content without being joined to a room. For instance, through user
+directories or invites.
+
+Clients can infer whether a reported event, room or user exists based on the
+404 responses of the reporting endpoints. Homeservers that wish to conceal
+this information MAY return 200 responses regardless of the existence of the
+reported subject.
```

Comment:
This has previously not been stated explicitly for the room- and event-reporting endpoints. I would consider it implied though, especially since the event-reporting endpoint covers the more sophisticated timing attack.

Reply:
This would require an MSC to change how the report room endpoint works, because the MSC was quite clear that it's a 404.